frontend/src/components/ModelInputs.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { ChangeEvent, useState } from "react";
+import { ModelConfig } from "../dto/FormConfig";
+import { Autocomplete, Button, TextField, Typography } from "@equinor/eds-core-react";
+
+const DEFAULTS = {
+    O2: 30,
+    H2O: 30,
+    H2S: 0,
+    SO2: 10,
+    NO2: 20,
+};
+
+const PPM_MAX = 1000000;
+
+function SubstanceAdder({ invisible, onAdd }: { invisible: string[]; onAdd: (subst: string) => void }) {
+    const [selected, setSelected] = useState<string | null>(null);
+
+    if (invisible.length === 0) return;
+    return (
+        <div style={{ display: "flex", alignItems: "center", marginTop: "1em" }}>
+            <Autocomplete
+                style={{ flexGrow: 1 }}
+                id="newConcentration"
+                label=""
+                placeholder="Add new"
+                options={invisible}
+                onOptionsChange={({ selectedItems }) => setSelected(selectedItems[0])}
+                selectedOptions={selected === null ? [] : [selected]}
+            />
+            <Button
+                onClick={() => {
+                    onAdd(selected!);
+                    setSelected(null);
+                }}
+                disabled={selected === null}
+            >
+                +
+            </Button>
+        </div>
+    );
+}
+
+function ParametersInput({
```

Comment:
I think this is more general than "ParametersInput". Technically this could be together with `ListWithItemAdder`, just without any items to add.. Would be nice not having to specialize these components..